src/dotnet/APIView/proposals/ProductCentricProposal.md

```diff
@@ -0,0 +1,306 @@
+# APIView Product-Centric Model Proposal
+
+## Overview
+
+This proposal introduces a new top-level **Product** entity to APIView. A Product represents a service across all languages and is always anchored to a TypeSpec specification. It aggregates:
+
+* The TypeSpec Review
+* All language-specific Reviews
+* Namespace approval history (approved + rejected)
+* Links to EngSys artifacts (tspconfig.yaml, metadata) and cross-language identifiers
+
+---
+
+## Problem Summary
+
+APIView’s current unit of organization is the **Review**, which corresponds to a single package in a single language. This structure does not provide:
+
+### 1. Cross-Language Grouping
+
+We cannot reliably visualize all languages for a service or know which SDKs exist or are missing.
+
+### 2. Early Namespace Approval
+
+Namespace approval depends on uploading SDK packages for each language, even though the only authoritative namespace source is **tspconfig.yaml**. We cannot approve namespaces until after SDK generation, which is often too late in the process.
+
+### 3. Namespace History
+
+APIView only tracks namespace approval at a *review* level. There is no historical record of:
+
+* Rejected namespaces
+* Withdrawn or replaced proposals
+* Namespace decisions tied to TypeSpec revisions
+
+### 4. Visibility for Stakeholders
+
+Stakeholders need one place to answer:
+
+* "What is the status of this service across languages?"
+* "Which namespaces were proposed and why were some rejected?"
+
+---
+
+## Goals
+
+The new **Product** model will:
+
+### ✓ Group all language reviews under a single entity
+
+Bound by the same cross-language identifier for a TypeSpec-defined service (nominally, that TypeSpec namespace).
+
+### ✓ Allow early namespace approval before any SDK is generated
+
+Using `tspconfig.yaml` as the single source of truth.
+
+### ✓ Track full namespace history
+
+Including rejected, withdrawn, and superseded proposals.
+
+### ✓ Improve stakeholder experience
+
+Navigation, dashboards, and discussions become cross-language by default.
+
+### ✓ Provide a stable home for TypeSpec reviews
+
+Every Product has exactly one TypeSpec Review.
```

Comment:
This can be confusing. One Azure service/product can have more than one typespec project.

Reply:
Renamed to Project.